.github/workflows/ci.yml

```diff
@@ -2,7 +2,6 @@ name: CI
 
 on:
   pull_request:
-    branches: ["main", "master"]
```

Comment:
I guess this was so the CI would trigger when merging on your feature branch? Could be put back with just main since we don't have a master branch

Reply:
This was fixed in #117